ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -765,20 +765,20 @@ void OwInterface::discard (double x, double y, double z, int id)
 }
 
 
-void OwInterface::deliverAction (int id)
+void OwInterface::taskDeliverSampleAction (int id)
```

Comment:
Another optional renaming.  The only thing that needs renaming is the ROS action and any references to that action name.

Reply:
I ended up keeping the optional renaming because it's consistent with the most recent command unification work (ie: armStow).